### PR TITLE
[[ Bug 16145]] Backport of PR #3679 to develop-6.7

### PR DESCRIPTION
**Pull request by livecodeian**
Original Pull Request: https://github.com/livecode/livecode/pull/3679

### CODE REVIEW

**Comment by peter-b** [comment]
@livecode-vulcan review ok 0dd6bfa

**Comment by livecode-vulcan** [comment]
💙 review by @peter-b ok 0dd6bfaa6bac0333dd62ccb3e8b298e758ef8ace

**Comment by livecode-vulcan** [comment]
😞 test failure 0dd6bfaa6bac0333dd62ccb3e8b298e758ef8ace
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/1982)
- try-community-mac: [pending](https://vulcan.livecode.com/builders/try-community-mac/builds/1995)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/1976)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/1988)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/1998)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/2004)
- try-community-android-armv6: [failure](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/2008)

File: engine/engine.gyp [hunk 1]
```diff
@@ -272,7 +272,7 @@
 							# Helpful for catching build problems
 							'-Wl,-no-undefined',
 							
-							'-Wl,-T,<(src_top_dir_abs)/engine/linux.link',
+							'-Wl,-T,<(src_top_dir_abs)/standalone-armv6-hf.link',
```
Comment (peter-b) [inline engine/engine.gyp]: I think there might be a missing `/engine` here?

Comment (livecodeian) [inline engine/engine.gyp]: Indeed, and also a missing standalone-armv6-hf.link file. Will fix in a moment.

**Comment by runrevmark** [comment]
@livecode-vulcan review ok 3c61a33

**Comment by livecode-vulcan** [comment]
💙 review by @runrevmark ok 3c61a3304eb58dcba0941ce8318f3d58d03f749a

**Comment by livecode-vulcan** [comment]
😎 test success 3c61a3304eb58dcba0941ce8318f3d58d03f749a
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/1987)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/2000)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/1981)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/1993)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/2003)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/2009)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/2013)

**Pull request merged**